content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -162,7 +171,61 @@ For details, see [Create a CoreWeave AI Object Storage bucket](https://docs.core
     }
     ```
 
-The clause beginning with `"Sid": "AllowUsersInOrg"` grants users in your W&B organization direct access to the bucket. If you don't need this ability, you can omit the clause from your policy.
+    **Dedicated Cloud** / **Self-Managed**:
+    
+    Replace `<cw-bucket>` with the CoreWeave bucket name and replace `<customer-namespace>` with the Customer Namespace value you obtained in step 2.
+
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+      {
+        "Sid": "AllowWandbUser",
+        "Action": [
+          "s3:GetObject*",
+          "s3:GetEncryptionConfiguration",
+          "s3:ListBucket",
+          "s3:ListBucketMultipartUploads",
+          "s3:ListBucketVersions",
+          "s3:AbortMultipartUpload",
+          "s3:DeleteObject",
+          "s3:PutObject",
+          "s3:GetBucketCORS",
+          "s3:GetBucketLocation",
+          "s3:GetBucketVersioning"
+        ],
+        "Effect": "Allow",
+        "Resource": [
+          "arn:aws:s3:::<cw-bucket>/*",
+          "arn:aws:s3:::<cw-bucket>"
+        ],
+        "Principal": {
+          "CW": "arn:aws:iam::wandb:static/wandb-integration"
```

Comment:
Could we have this in the same style as the AWS docs? where the principal is 
`"CW": "arn:aws:iam::wandb:static/<wandb_coreweave_principal>"` 
and `<customer-namespace> vs <wandb-org-id>`
so we don't have to have 2 separate policies? 

<img width="1139" height="692" alt="Screenshot 2025-08-28 at 14 07 43" src="https://github.com/user-attachments/assets/2c99785f-97ae-45f2-beb5-c465a078bb9f" />